cms/apps/media/models.py

```diff
@@ -76,7 +90,15 @@ def __str__(self):
         return self.title
 
     class Meta:
-        ordering = ("title",)
+        ordering = ['title']
+
+    def save(self, force_insert=False, force_update=False, using=None, update_fields=None):
+        super(File, self).save(force_insert, force_update, using, update_fields)
+
+        self.width = self.get_width()
```

Comment:
I would also test what happens if you upload something that's not an image, like a PDF or a .docx file, just to make sure we're covering those cases.